lib/internal/main/worker_thread.js

```diff
@@ -84,9 +84,14 @@ if (process.env.NODE_CHANNEL_FD) {
     workerThreadSetup.unavailable('process.disconnect()');
 }
 
+let cachedCwd = '';
+let lastCounter = -1;
+const originalCwd = process.cwd;
```

Comment:
Can we put these right before the override function is defined? I think that would be a good idea for readability